js/util/RemoteController.js

```diff
@@ -2,13 +2,24 @@ import { Vector3 } from 'https://cdn.skypack.dev/pin/three@v0.137.0-X5O2PK3x44y1
 import localProxy from "./localProxy.js";
 import goLivePanel from '../UiElements/goLivePanel.js';
 import peerIdDisplay from '../UiElements/peerIdDisplay.js';
-import chatBox from '../UiElements/chatBox.js';
+import {chatBox, toggleConnectRoom} from '../UiElements/chatBox.js';
 import friendManagement from '../UiElements/buttons/friendManagement.js';
 import { AvatarController } from '../entities/AvatarController.js';
 import { toggleCallBox, callBox, addCamera } from '../UiElements/callBox.js';
+import { PeerGroup } from './peerjs-groups.js';
+import alertBox from '../UiElements/alertBox.js';
 
 // PeerJs is injected in the window
 const Peer = window.Peer;
+const SIGNALLING_OPTIONS =  {
+    secure: 'wss://',
+    host: 'spacetime-peerserver.herokuapp.com',
```

Comment:
Hi @codeeMadness could you please open a new pr to include the server side code into the repository.

This repository is public, be careful not to include any private key or token.

thank you